services/mock-blockchain/README.md

```
@@ -0,0 +1,396 @@
+
+## Development
+
+To contribute to the mock blockchain services:
+
+1. Fork the repository
+2. Create a feature branch: `git checkout -b feature/my-feature`
+3. Make your changes
+4. Run tests: `bun test`
+5. Commit your changes: `git commit -m 'Add some feature'`
+6. Push to the branch: `git push origin feature/my-feature`
+7. Submit a pull request
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)